### DISCUSSION TITLE
为什么训练完之后总会出现2023/09/28 04:07:35 - mmengine - ERROR - /home/jupyter/miniconda3/envs/mmyolo/lib/python3.8/site-packages/mmdet/evaluation/metrics/coco_metric.py - compute_metrics - 461 - The testing results of the whole dataset is empty.

### DISCUSSION THREAD

**Discussion by chenhan1hao** [open-mmlab/mmyolo#903] [Q&A]
为什么训练完之后总会出现2023/09/28 04:07:35 - mmengine - ERROR - /home/jupyter/miniconda3/envs/mmyolo/lib/python3.8/site-packages/mmdet/evaluation/metrics/coco_metric.py - compute_metrics - 461 - The testing results of the whole dataset is empty.

**Comment by chengyuanteng** [comment]
解决了吗，我也是
